academy/3-running-a-chain/node-api-and-cli.md

```diff
@@ -217,25 +215,23 @@ $ cat ~/.simapp/config/genesis.json
 
 <HighlightBox type="tip">
 
-It helps to have the concepts very clear in mind when working hands-on with the Cosmos SDK. Need a refresher? Dive right into the [section on _Accounts_ in the _Main Concepts_ chapter](../2-main-concepts/accounts.md).
+To take a step back and revisit how accounts work, go to the [section on _Accounts_ in the _Main Concepts_ chapter](../2-main-concepts/accounts.md).
 
 </HighlightBox>
 
-You can also inspect your keys. These are held in the backend keyring, which by default is that of the operating system:
+You can also inspect your keys. These are held in the backend keyring which by default is that of the operating system:
 
 ```sh
 $ ./simd keys list
 []
 ```
 
-As you might have expected, you do not have any keys yet.
-
-Fix that and add a new key:
+First add a new key:
 
 ```sh
-$ ./simd keys add b9lab
+$ ./simd keys add academy
```

Comment:
Again, I would suggest moving the output from the command to a different code snippet for copy-pasting purposes.